test/data_bundle_test.dart

```diff
@@ -23,7 +26,18 @@ void main() async {
       await dataItem.sign(wallet);
 
       expect(await dataItem.verify(), isTrue);
+    });
 
+    test('comfirm data item with wrong signaure fails verify', () async {
```

Comment:
What qualifies as a `wrong signature`? How does your test case result in a wrong signature?

Reply:
Tags are a part of deriving the signature. What we are doing in this test is to create a data item and sign it. Then we append a new tag to the same data item. When we verify again the tags in the previous signature and the newly derived signature won't match. So it fails